modules/apps/portal-search/portal-search-api/src/main/resources/content/Language_es.properties

```diff
@@ -16,7 +16,7 @@ index-status-manager-configuration-name=Administrador de estado de índice
 index-writer-helper-configuration-name=Asistente de escritura de indexación
 indexer-registry-configuration-name=Registro del índice
 indexing-batch-sizes=Indexando tamaños de lotes
-indexing-batch-sizes-help=El número de documentos que se indizará por lote para los tipos de modelo que admiten la indexación por lotes. El valor por defecto es 10 000. Para modelos con documentos de gran tamaño, si se reduce este valor, puede mejorarse la estabilidad al ejecutar una nueva indexación completa.
+indexing-batch-sizes-help=El número de documentos que se indizará por lote para los tipos de modelo que admiten la indexación por lotes. El valor por defecto es 10000. Para modelos con documentos de gran tamaño, si se reduce este valor, puede mejorarse la estabilidad al ejecutar una nueva indexación completa.
```

Comment:
Removed `0xa0` (non-breaking space) rather than using `0x20` (space), as there is no thousands delimiter in English. I didn't check other locals for consistency.